assets/src/edit-story/components/keyboardShortcutsMenu/landmarkShortcuts.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { Fragment } from 'react';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { Text, THEME_CONSTANTS } from '../../../design-system';
+import useKeyboardShortcuts from './getKeyboardShortcuts';
+import ShortcutLabel from './shortcutLabel';
+
+const LandmarksWrapper = styled.dl`
+  background-color: ${({ theme }) => theme.colors.bg.secondary};
+  margin: 0;
+  padding: 16px 24px;
+  display: flex;
+  justify-items: space-between;
```

Comment:
I think this is what you were going for. Should remove the need for that space element & clean up the markup a bit
```suggestion
  justify-content: space-between;
```

Reply:
Fixed in f3af403d2